pyproject.toml

```diff
@@ -0,0 +1,109 @@
+[build-system]
+requires = ['hatchling', 'hatch-vcs']
+build-backend = 'hatchling.build'
+
+[project]
+name = 'fmf'
+authors = [
+    { name = 'Petr Splichal', email = 'psplicha@redhat.com' },
+]
+maintainers = [
+    { name = 'Petr Splichal', email = 'psplicha@redhat.com' },
+]
+description = 'Flexible Metadata Format'
+readme = 'README.rst'
+license = 'GPL-2.0-or-later'
+license-files = { paths = ['LICENSE'] }
+requires-python = '>=3.9'
+classifiers = [
+    'Natural Language :: English',
+    'Programming Language :: Python :: 3.9',
+    'Programming Language :: Python :: 3.10',
+    'Programming Language :: Python :: 3.11',
+    'Programming Language :: Python :: 3.12',
+    'Topic :: Utilities',
+]
+keywords = [
+    'metadata',
+    'testing',
+]
+dependencies = [
+    'ruamel.yaml',
+    'filelock',
+    'jsonschema',
+]
+dynamic = ['version']
+
+[project.urls]
+Homepage = 'https://github.com/teemtee/fmf'
+Documentation = 'https://fmf.readthedocs.io'
+
+[project.optional-dependencies]
+tests = [
+    'pytest',
+]
+tests-cov = [
+    'fmf[tests]',
+    'pytest-cov',
+]
+docs = [
+    'sphinx',
+    'sphinx_rtd_theme',
+]
+
+[project.scripts]
+fmf = 'fmf.cli:cli_entry'
+
+[tool.hatch]
+version.source = 'vcs'
+
+[tool.hatch.build.targets.wheel]
+packages = ['fmf']
```

Comment:
Isn't this redundant?

Reply:
I don't remember, I usually use src-layout where this is indeed automatically filtered. The purpose of that is to filter any packages that are not `fmf` like `tests`, so we want to avoid any potential auto-detection there

Reply:
Make sense. Do you want me to test if there is any difference?

Reply:
Would be awesome :), otherwise I will do it also this weekend

Reply:
Confirmed it works for the current structure